src/common/models/abilities.ts

```diff
@@ -7,37 +7,37 @@ export interface Ability {
   label: string;
   proficient?: boolean;
   id: string;
-  category: ModifierTypes;
+  category: string;
 }
 
 
 
-export const abilities:{label: string, category: ModifierTypes}[] = [
-  {label: "Acrobatics", category: ModifierTypes.DEX},
-  {label: "AnimalHandling", category: ModifierTypes.WIS},
-  {label: "Arcana", category: ModifierTypes.INT},
-  {label: "Athletics", category: ModifierTypes.STR},
-  {label: "Deception", category: ModifierTypes.CHA},
-  {label: "History", category: ModifierTypes.INT},
-  {label: "Insight", category: ModifierTypes.WIS},
-  {label: "Intimidation", category: ModifierTypes.CHA},
-  {label: "Investigation", category: ModifierTypes.INT},
-  {label: "Medicine", category: ModifierTypes.WIS},
-  {label: "Nature", category: ModifierTypes.INT},
-  {label: "Perception", category: ModifierTypes.WIS},
-  {label: "Performance", category: ModifierTypes.CHA},
-  {label: "Persuasion", category: ModifierTypes.CHA},
-  {label: "Religion", category: ModifierTypes.INT},
-  {label: "SleightOfHand", category: ModifierTypes.DEX},
-  {label: "Stealth", category: ModifierTypes.DEX},
-  {label: "Survival", category: ModifierTypes.WIS},
+export const abilities:{label: string,id:string; category: string}[] = [
+{label: "Acrobatics", id: "Acrobatics", category: ModifierTypes.DEX},
+{label: "Animal Handling", id: "AnimalHandling", category: ModifierTypes.WIS},
+{label: "Arcana", id: "Arcana", category: ModifierTypes.INT},
+{label: "Athletics", id: "Athletics", category: ModifierTypes.STR},
+{label: "Deception", id: "Deception", category: ModifierTypes.CHA},
+{label: "History", id: "History", category: ModifierTypes.INT},
+{label: "Insight", id: "Insight", category: ModifierTypes.WIS},
+{label: "Intimidation", id: "Intimidation", category: ModifierTypes.CHA},
+{label: "Investigation", id: "Investigation", category: ModifierTypes.INT},
+{label: "Medicine", id: "Medicine", category: ModifierTypes.WIS},
+{label: "Nature", id: "Nature", category: ModifierTypes.INT},
+{label: "Perception", id: "Perception", category: ModifierTypes.WIS},
+{label: "Performance", id: "Performance", category: ModifierTypes.CHA},
+{label: "Persuasion", id: "Persuasion", category: ModifierTypes.CHA},
+{label: "Religion", id: "Religion", category: ModifierTypes.INT},
```

Comment:
Added a label, didn't want to deal with figuring out how to break apart the word.